adventure/adventure.js

```diff
@@ -0,0 +1,53 @@
+import loadProfile from '../common/load-profile.js';
+import { getAlly, saveAlly } from '../data/save-ally.js';
+import { adventures } from '../data/meta-data.js';
+import createChoice from './create-choice.js';
+import findById from '../common/find-by-id.js';
+import scoreAdventure from './score-adventure.js';
+
```

Comment:
your file paths for loadProfile and findyId are wrong.  Looks like they live in your data folder and not your common folder.